packages/checkout/sdk/src/widgets/definitions/configurations/widget.ts

```diff
@@ -7,4 +7,5 @@ import { WidgetTheme } from './theme';
 export type WidgetConfiguration = {
   /** The theme of the Checkout Widget (default: "DARK") */
   theme?: WidgetTheme;
+  language?: 'en' | 'ja' | 'ko' | 'zh'
```

Comment:
is this the same as `LanguageChanged`?

Reply:
Yes good catch, I've improved that and mapped back to WidgetLanuage.